dc_eval_models/test_dc_eval.py

```diff
@@ -0,0 +1,14 @@
+from bert_base import BertBaseModel
+from deberta_base import DebertaBaseModel
+
+dc_text = "Making choices in life can be very difficult. People often ask for advice when they can not decide on one thing. It's always good to ask others for their advice when making a choice. When you have multiple opinions you have the ability to make the best choice for yourself."
+dc_type = "Lead"
+essay = open('./dc_eval_models/sample_essay.txt', 'r').read()
+
+bert_base = BertBaseModel()
+bert_res = bert_base.predict(dc_text, dc_type, essay)
+print('BERT result:', bert_res)
+
+deberta_base = DebertaBaseModel()
+deberta_res = deberta_base.predict(dc_text, dc_type, essay)
+print('DeBERTa result:', deberta_res)
```

Comment:
Having tests is good. However, 
- if the goal is to make sure the code works, it should be made as unit tests,
- if the goal is to show examples of how to use the code, it should be in the `__main__` function of the corresponding file,
- if the goal is for personal use only, it should be committed before committing.